db/migrate/20200212150214_add_key_info_to_schools.rb

```diff
@@ -0,0 +1,17 @@
+class AddKeyInfoToSchools < ActiveRecord::Migration[5.2]
+  def change
+    add_column :schools, :status, :string
```

Comment:
Wow. These are a lot of columns to add at once. 😄

Reply:
It's based on the columns needed for bigquery reporting and those needed for the key information tickets, wanted to avoid performing multiple migrations if possible 😄